torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -127,6 +146,7 @@ void LowerAllTuples(std::shared_ptr<Graph>& graph) {
 static void LowerSimpleTuples(Block* block) {
   for(auto n : block->nodes()) {
     // make any TupleUnpack dead by undoing TupleUnpack(TupleConstruct())
+    // and TupleIndex(TupleIndex)
```

Comment:
I'm confused what is this comment supposed to mean. You can't flatten two levels of indexing into a single one. Did you want to say `TupleIndex(TupleConstruct())`?

Reply:
Yes that's what i meant